destination.go

```diff
@@ -66,6 +66,13 @@ func (d *Destination) Open(ctx context.Context) (err error) {
 	}
 	sdk.Logger(ctx).Debug().Msgf("opened channel")
 
+	// We setup the channel in confirm mode, so that we ensure the "written"
+	// total of records that we return is correct.
+	noWait := false
+	if err := d.ch.Confirm(noWait); err != nil {
```

Comment:
Why not sending `d.ch.Confirm(false)`

Reply:
Never mind. I think it reads better as it is since the API is not very intuitive with the double negation.